backend/scheduler/metric_scheduler.py

```
@@ -0,0 +1,201 @@
+        """스케줄러 시작"""
+        self.running = True
+        print(f"[스케줄러] 시작됨 (실행 주기: {self.interval_minutes}분)")
+        
+        while self.running:
+            try:
+                await self.execute_all_metrics()
+                await asyncio.sleep(self.interval_minutes * 60)
+            except KeyboardInterrupt:
+                print("\n[스케줄러] 중지 신호 수신")
+                break
+            except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)